docs/src/pages/guides/typescript/typescript.md

```diff
@@ -2,7 +2,7 @@
 
 <p class="description">You can add static typing to JavaScript to improve developer productivity and code quality thanks to TypeScript.</p>
 
-Have a look at the [Create React App with TypeScript](https://github.com/mui-org/material-ui/tree/master/examples/create-react-app-with-typescript) example. A minimum version of TypeScript 2.8 is required.
+Have a look at the [Create React App with TypeScript](https://github.com/mui-org/material-ui/tree/master/examples/create-react-app-with-typescript) example. A minimum version of TypeScript 3.2 is required.
```

Comment:
How about the following?

```suggestion
Material-UI requires a minimum version of TypeScript 3.2.

Have a look at the [Create React App with TypeScript](https://github.com/mui-org/material-ui/tree/master/examples/create-react-app-with-typescript) example.
```

Or maybe the minimum version should go into https://material-ui.com/getting-started/supported-platforms?

Reply:
> Or maybe the minimum version should go into https://material-ui.com/getting-started/supported-platforms?

We have already mentioned this in the past, much agree 👍

Reply:
To summarise, do you want the minimum version to be 3.2 on both pages?

Reply:
I guess it would help too?